src/main/java/org/sagebionetworks/web/client/presenter/EntityPresenter.java

```diff
@@ -331,24 +320,23 @@ public Widget asWidget() {
 
   @EventHandler
   public void onEntityUpdatedEvent(EntityUpdatedEvent event) {
-    List<SynapseReactClientEntityQueryKey> queryKey =
-      SynapseReactClientEntityQueryKey.create(
-        QueryKeyConstants.ENTITY,
-        event.getEntityId()
-      );
-    queryClient.resetQueries(queryKey);
+    KeyFactory keyFactory = new KeyFactory(
+      authenticationController.getCurrentUserAccessToken()
+    );
+
+    List<?> queryKey = keyFactory.getEntityQueryKey(event.getEntityId());
+    queryClient.invalidateQueries(queryKey);
     globalAppState.refreshPage();
   }
```

Comment:
This invalidation doesn't include the token in the query key, so who knows how long it's been broken...let's use `KeyFactory` to reliably generate the correct key.